src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -305,15 +525,15 @@ shopCheckout() {
 }
 ```
 
-This example will send a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) `shopCheckout` to Sentry, containing all outgoing requests that happen in `validateShoppingCartOnServer` as spans. The transaction will also contain a `task` span that measures how long `processAndValidateShoppingCart` took. Finally, the call to `ApmIntegrations.Tracing.finshIdleTransaction()` will finish the [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) and send it to Sentry. Calling this is optional; if it is not called, the integration will automatically send the [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) itself after the defined `idleTimeout` (default 500ms).
+This example will send a transaction `shopCheckout` to Sentry, containing all outgoing requests that happen in `validateShoppingCartOnServer` as spans. The transaction will also contain a `task` span that measures how long `processAndValidateShoppingCart` took. Finally, the call to `ApmIntegrations.Tracing.finishIdleTransaction()` will finish the transaction and send it to Sentry. Calling this is optional; if it is not called, the integration will automatically send the transaction itself after the defined `idleTimeout` (default 500ms).
 
 **What is an activity?**
 
 The concept of an activity only exists in the `Tracing` integration in JavaScript Browser. It's a helper that tells the integration how long it should keep the `IdleTransaction` alive. An activity can be pushed and popped. Once all activities of an `IdleTransaction` have been popped, the `IdleTransaction` will be sent to Sentry.
 
 **Manual Transactions**
 
-To manually instrument a certain region of your code, you can create a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) to capture it.
+To manually instrument a certain region of your code, you can create a transaction to capture it.
```

Comment:
same as above `item.getTransaction()` is very hand wavy as to what one passes into `transaction`. And given that the function name is `startSpan` implies that there's a transaction object to pass around

Reply:
Same as above. Stay tuned!